go/api/generic_commands.go

```diff
@@ -30,4 +30,65 @@ type GenericBaseCommands interface {
 	//
 	// [valkey.io]: https://valkey.io/commands/del/
 	Del(keys []string) (Result[int64], error)
+
+	// Unlink (delete) multiple keys from the database. A key is ignored if it does not exist.
+	// This command, similar to Del However, this command does not block the server
```

Comment:
```suggestion
	// This command, similar to `Del`, however, this command does not block the server
```
Does go docs support links? If yes, please add.

Reply:
Hi @Yury-Fridlyand - I included valkey site command doc link for each commands
![image](https://github.com/user-attachments/assets/9e948e52-45ff-4454-a5e0-414caf9bde32)

Reply:
Hi @Yury-Fridlyand - you can now check it on my latest commit, if this is what your looking for. :)